Gelst 13

gelst13 opened the pull request:
added part (4 methods) that 1)extracts time from description, 2)adds year to date string and 3)returns date of event as timestamp

gelst13 committed:
changes by Gelst13

gelst13 committed:
Merge branch 'main' into gelst_13

# Conflicts:
#	all.py

gelst13 committed:
changes by Gelst13

hedgehogoutfit commented on all.py:
```diff
@@ -14,7 +14,7 @@
 def get_posts(group_id, n):  # n is amount of posts
     """Return list of n dictionaries with posts"""
-    session = vk_api.VkApi(token='')
+    session = vk_api.VkApi(token='250a634d14a734f4e6a2422fd581ef4d3aabcd51646097676a6b4fac22baa74831ef5aec48ccbe4009706')
```
are you sure about that commit?)

hedgehogoutfit commented on all.py:
```diff
@@ -46,23 +47,58 @@ def extract_text_link(advert, group_id):
-    place = clean_text(search_place(text_after_date))
-    address = clean_text(search_address(text_after_date))
-    return date_, description, place, address, link
+    else:
```
думаю else тут лишний. без него будет то же самое

hedgehogoutfit committed:
Update all.py

gelst13 committed:
Merge pull request #3 from hedgehogoutfit/hedgehogoutfit-patch-1

Update all.py

gelst13 committed:
first version of supporting modules:
insertings - to save data into sql .db
select_event - 1 type of select "filter" + write into .html
drop_duplicate_rows - tested only on "prototype .db"
